pkg/controller/utils/utils.go

```diff
@@ -38,3 +45,29 @@ func MarshalIAMPolicy(role awsv1alpha1.AWSFederatedRole) (string, error) {
 
 	return string(jsonPolicyDoc), nil
 }
+
+// NewAccountForCR returns a Pending, Unclaimed CR with a name of libra-ops-<generated-string>
+func NewAccountForCR(namespace string) *awsv1alpha1.Account {
```

Comment:
@fahlmant I think we could have a better function name? How does this return a Pending CR?

Reply:
Fixing